pallets/pallet-did-lookup/src/migrations.rs

```diff
@@ -28,7 +28,9 @@ use frame_support::{
 use sp_std::{marker::PhantomData, vec};
 
 #[cfg(feature = "try-runtime")]
-use frame_support::traits::OnRuntimeUpgradeHelpersExt;
+use frame_support::codec::{Decode, Encode};
```

Comment:
I guess these can be combined?

Reply:
fde113cb1b81146770db9d3edfab89e40100617a